examples/heartrate_monitor_peripheral.py

```diff
@@ -0,0 +1,117 @@
+"""Example of how to create a Peripheral device/GATT Server"""
+# Standard modules
+import logging
+import struct
+
+# Bluezero modules
+from bluezero import async_tools
+from bluezero import adapter
+from bluezero import peripheral
+
+HRM_SRV = '0000180D-0000-1000-8000-00805f9b34fb'
+HR_MSRMT_UUID =      '00002a37-0000-1000-8000-00805f9b34fb'
+BODY_SNSR_LOC_UUID = '00002a38-0000-1000-8000-00805f9b34fb'
+HR_CTRL_PT_UUID =    '00002a39-0000-1000-8000-00805f9b34fb'
+
+
+heartrate = 60
+def read_heartrate():
+    global heartrate
```

Comment:
Does this need to be a global?
I think the value is only used in the function so the global could be removed?

Reply:
Pending the results of script vs class, I would champion the global as the simplest design pattern for a new programmer to follow for this variable that increments each time the read_heartrate function is called.  In C++ this would be a function static variable.